code/createRandomImages/app.js

```diff
@@ -0,0 +1,20 @@
+// event listener
+
+
+
+let width = document.querySelector(".width").value;
+let height = document.querySelector('.height').value;
+
+let newImage = document.createElement('img') ;
+
+const createImage = function () {
+   let url = 'https://source.unsplash.com/random/' + width +'/'+ height + '?' + Date();
```

Comment:
I am not suer Why you add Date() in the end?
```suggestion
   let url = 'https://source.unsplash.com/random/' + width +'x'+ height;
```
This code should give you What you want.